tests/integration/api_v3/rpc/dryRunTransactions.test.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { VALID_TRANSACTION, INVALID_TRANSACTION } = require('../constants/dryRunTransactions');
+
+const {
+	request,
+} = require('../../../helpers/socketIoRpcRequest');
+
+const {
+	invalidParamsSchema,
+	jsonRpcEnvelopeSchema,
+	serverErrorSchema,
+} = require('../../../schemas/rpcGenerics.schema');
+
+const {
+	dryrunTransactionSchema,
+	goodRequestSchema,
+	metaSchema,
+} = require('../../../schemas/api_v3/dryrunTransaction.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v3`;
+const postDryrunTransaction = async params => request(wsRpcUrl, 'post.transactions.dryrun', params);
+
+describe('Method post.transactions.dryrun', () => {
+	it('Post dryrun transaction succesfully', async () => {
```

Comment:
Could also add another test case where we dry run an existing transaction? The expected `response.data` should look something like:
```
{
	 success: false,
	 events: [],
}
```